### ISSUE TITLE
Optimize multiserver script deployment

### ISSUE THREAD

**Issue by thomasjammet** [thomasjammet/dev-notes#1]
In a multiserver deployement the main worry is to get not incremental complexity for every server addition on the cluster. A substantive work can be done to optimize exchange between every server, and tries to keep their complexity independant of the number of server on the cluster.
To illustrate this problematic, look what happens on a unknown P2P attempt with current scripts:

``` lua
function onRendezVousUnknown(protocol,peerId)
      return mona.servers -- redirect to all the connected servers
end
```

The client searchs the peer identified by “peerId” which is unknown for this server, and the server redirects the client in giving all the address of the other servers.
The client will now required in same time (parallel) ALL the other servers! If the cluster contains many servers, it can obstruct all the cluster just for one client which search one other (and it’s a common way which happens often).
In conclusion,with the current script, if 1xMona supports 20 000 clients, 2xMona will support less than 40 000 clients, and 3xMona will support really very less than 60 000 clients.
